UI/src/view/csvReportsWorkList.js

```diff
@@ -0,0 +1,489 @@
+              '<div class="form-group" disabled="disabled" >' +
+                '<div class="row">' +
+                  '<div class="col-xs-5">' +
+                    '<select id="municipalities_search" class="form-control" multiple="multiple">' +
+                    '</select>' +
+                  '</div>' +
+                  '<div class="col-xs-2">' +
+                    '<button type="button" id="municipalities_search_rightSelected" class="action-mode-btn btn btn-block edit-mode-btn btn-primary"> &gt; </button>' +
+                    '<button type="button" id="municipalities_search_leftSelected" class="action-mode-btn btn btn-block edit-mode-btn btn-primary"> &lt; </button>' +
+                  '</div>' +
+                  '<div class="col-xs-5">' +
+                    '<select name="municipalityNumbers" id="municipalities_search_to" class="form-control" multiple="multiple">' +
+                    '</select>' +
+                  '</div>' +
+                '</div>' +
+              '</div>' +
+            '</div>' +
+            '<div class="form-group">' +
+              '<div class="form-group" disabled="disabled" >' +
+                '<div class="row">' +
+                  '<div class="col-xs-5">' +
+                    '<select id="assets_search" class="form-control" multiple="multiple">' +
+                    '</select>' +
+                  '</div>' +
+                  '<div class="col-xs-2">' +
+                    '<button type="button" id="assets_search_rightSelected" class="action-mode-btn btn btn-block edit-mode-btn btn-primary"> &gt; </button>' +
+                    '<button type="button" id="assets_search_leftSelected" class="action-mode-btn btn btn-block edit-mode-btn btn-primary"> &lt; </button>' +
+                  '</div>' +
+                  '<div class="col-xs-5">' +
+                    '<select name="assetNumbers" id="assets_search_to" class="form-control" multiple="multiple">' +
+                    '</select>' +
+                  '</div>' +
+                '</div>' +
+              '</div>' +
+            '</div>' +
+            '<div class="form-controls" style="text-align: right;">' +
+              '<button id="send_btn" class="btn btn-primary btn-lg" disabled="disabled">Luo raportti</button>' +
+            '</div>' +
+          '</div>'+
+          '<div class="job-status">' +
+          '</div>'+
+          '<div class="job-content">'+
+          '</div>';
+
+      return form;
+    };
+
+
+    var refreshJobs = function() {
+      var jobsInProgress = $('.in-progress').map(function(){
+        return $(this).attr('id');
+      });
+
+      if(!_.isEmpty(jobsInProgress)) {
+        backend.getExportsJobsByIds(jobsInProgress.toArray()).then(function(jobs){
+          var endedJobs = _.filter(jobs, function(job){return job.status !== 1;});
+          _.map(endedJobs, replaceRow);
+        });
+      } else {
+        clearInterval(refresh);
+        refresh = null;
+      }
```

Comment:
There is no need to negate the _.isEmpty
```javascript
     if(_.isEmpty(jobsInProgress)) {
        clearInterval(refresh);
        refresh = null;
      } else {
        backend.getExportsJobsByIds(jobsInProgress.toArray()).then(function(jobs){
          var endedJobs = _.filter(jobs, function(job){return job.status !== 1;});
          _.map(endedJobs, replaceRow);
        });
      }
```

Reply:
don't understand the advantage on that approach or leave as is